### PR TITLE
refactor logging

### PR DESCRIPTION
**Pull request by mLuby**
@armandopmj take a look at [config/log.js](https://github.com/projectLiftOff/surpriseHer.server/pull/14/files#diff-069c2e3498819978b3f08052ea769309R1) (mostly the same as winstonLogger.js, which it replaced).
Then I used that `log` object in the rest of the code.

Let me know if you have comments or I should merge it (or you can).

### CODE REVIEW

**Comment by mLuby** [comment]
fixes #2

**Comment by armandopmj** [comment]
@mLuby will review shortly

File: server/services/twilio/twilio.main.js [hunk 1]
```diff
@@ -8,18 +8,19 @@ function send( to, from, body, attempts ) {
         to: '+1' + to,
         from: '+1' + from,
         body: body
-    }, function( err, data ){
-        if( err ) {
+    }, function( error, data ){
+        if (error) {
+            log.error('Sending Monthly Options text using twilio falied', {error}, {to, from, message: body, resendAttempts: attempts} );
```
Comment (armandopmj) [inline server/services/twilio/twilio.main.js]: @mLuby in the last object the `to` and `from` vars are stings which will be turned into keys but they have no value assigned... (e.g. `, key: value,` vs `, key,`) wont this break?

File: server/services/twilio/twilio.main.js [hunk 2]
```diff
@@ -8,18 +8,19 @@ function send( to, from, body, attempts ) {
...
             if( attempts < 3 ) {
                 attempts++;
                 send( to, from, body, attempts );
             }
-            else log.error( 'Sending Monthly Options text using twilio falied', {to: to, from: from, message: body, resendAttempts: attempts} );
         }
         else {
             // TODO: log successful message sent in db
-            log.info( 'Monthly Options text message was sent successfuly', {to: to, from: from, message: body, resendAttempts: attempts} )
+            log.info( 'Monthly Options text message was sent successfuly', {to, from, message: body, resendAttempts: attempts} )
```
Comment (armandopmj) [inline server/services/twilio/twilio.main.js]: @mLuby same issue as above

File: server/services/twilio/twilio.main.js (hunk 1 above)
Comment (mLuby) [inline server/services/twilio/twilio.main.js]: Oh, that's ES6, which I've been trying to practice, but is optional at this point. If you feel strongly about using it or not, I'm fine with going either way. 
```
const x = 2
function z () { return false }
var es5 = {
  x: x,
  y: function (n) { return Math.random() > n }
  z: z
}
let es6 = {
  x, 
  y: function (n) { return Math.random > n },
  z
}
```

Comment (armandopmj) [inline server/services/twilio/twilio.main.js]: @mLuby ah nice! no lets use it, I could use the exposure and practice

**Comment by mLuby** [comment]
@armandopmj okay so we'll head toward ES6. Okay to merge this or are you still reviewing?

**Pull request merged**